src/resolver/forge/adapter/ForgeGradle3.resolver.ts

```diff
@@ -31,7 +31,7 @@ export class ForgeGradle3Adapter extends ForgeResolver {
         if(version.getMinor() === 12 && VersionUtil.isOneDotTwelveFG2(libraryVersion)) {
             return false
         }
-        return VersionUtil.isVersionAcceptable(version, [12, 13, 14, 15, 16])
+        return VersionUtil.isVersionAcceptable(version, [12, 13, 14, 15, 16, 17, 18, 19])
```

Comment:
```suggestion
        return VersionUtil.isVersionAcceptable(version, [12, 13, 14, 15, 16])
```
Adding other versions would be better in another PR